knowledge-base/basic-authentication-with-fiddler-everywhere.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Basic Authentication with Fiddler Everywhere
+description: This article briefs about the steps of basic authentication with fiddler everywhere.
+type: how-to
+slug: basic-authentication-with-fiddler-everywhere
+tags: basic authentication, fiddler everywhere, authentication with fiddler everywhere
+publish: true
+res_type: kb
+---
+
+## Environment
+
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>1.0.1</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Fiddler Everywhere</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+Fiddler Everywhere certificate inspects network traffic and debugs it. The __Composer__ feature in Fiddler Everywhere enables you to manually build and send HTTP, HTTPS requests. Click [here]({%slug composer%}) to know more about Fiddler Everywhere Composer.
```

Comment:
1) "Fiddler Everywhere certificate inspects network traffic and debugs it" - this sentence sounds very strange. The certificate inspects and debugs the network traffic? Perhaps the certificate "allows the inspection and debugging of network traffic"?

2) build and send HTTP, HTTPS requests. -> build and send HTTP **or** HTTPS requests.

3) "learn" sounds better than "know" in the last sentence.